backend/tests/projects/toolmodels/conftest.py

```diff
@@ -58,6 +58,11 @@ def fixture_git_type(request: pytest.FixtureRequest) -> git_models.GitType:
     return request.param
 
 
+@pytest.fixture(name="status", params=[200])
+def fixture_status(request: pytest.FixtureRequest) -> int:
+    return request.param
```

Comment:
Please use a more descriptive name for it, for example `github_response_status`.

Also, it's only used in `test_diagrams.py`, so I'd also move it into the file.